src/Controller/CRUDController.php

```diff
@@ -985,8 +985,6 @@ public function aclAction($id = null)
             throw $this->createNotFoundException('ACL are not enabled for this admin');
         }
 
-        $id = $request->get($this->admin->getIdParameter());
```

Comment:
This cannot be removed because it gets the right `id`, let's say you have a parent - child association, the url would be something like:

`/group/{id}/user/{childId}/acl`

In this case you want the `$id` to be the value of the `childId` placeholder. There is a recent discussion in https://github.com/sonata-project/SonataAdminBundle/pull/5807 with more details.